lit-next.code-workspace

```diff
@@ -20,6 +20,10 @@
       "name": "ssr",
       "path": "packages/labs/ssr"
     },
+    {
```

Comment:
VS Code has gotten a lot better about working with nested packages that aren't workspace roots, so I don't know if we need to add every package here anymore. I'm also fine with it.

Reply:
+1, but not in this PR